client/src/components/styleNames.jsx

```diff
@@ -0,0 +1,25 @@
+// Style and Selected Style
+// How will you display the style name?
+// How will you label the style so users know it's the style?
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StyleName = ({ styles }) => (
+  <div>
+    <h3 className="styles-name">
+      Style →&nbsp;
```

Comment:
Is this little arrow an emoji? Hard to see on Github. Curious what it's pointing to

Reply:
It is an arrow icon and `&nbsp` means a non-breaking space, essentially I needed a single space between the arrow and the category name and didn't want to have to use a string 😆